scene/gui/rich_text_label.cpp

```diff
@@ -1179,7 +1179,8 @@ void RichTextLabel::add_text(const String& p_text) {
 			item->line=current_frame->lines.size();
 			_add_item(item,false);
 			current_frame->lines.resize(current_frame->lines.size()+1);
```

Comment:
`rich_text_label.cpp` is really complicated for me, but I guess this line should be outside of `if` to increase position of line.